src/partials/proposal-to-leave/proposal-to-leave.html

```diff
@@ -0,0 +1,107 @@
+<div class="container proposal-to-leave">
+  <h1>
```

Comment:
We already use `<h1>` on the Hero section. Please use `<h2 >` here, and add class for styling